content/docs/command-reference/studio/login.md

```diff
@@ -40,6 +40,13 @@ and assigns a random name as the token name.
 
 ## Available scopes
 
-- `EXPERIMENTS` - Experiment operations
-- `DATASETS` - Dataset operations
-- `MODELS` - Model registry operations
+- `EXPERIMENTS` - Experiment operations. DVC uses this to share [live
+  experiments] and to notify [Studio](https://studio.iterative.ai/) about
+  [pushed experiments]
+- `MODELS` - Model registry operations like downloading model using
+  `dvc artifacts get`
+- `DATASETS` - Dataset operations. [Coming soon](https://cloud.dvc.ai).
```

Comment:
@amritghimire I meant let's move this to https://dvc.org/doc/studio/user-guide/account-and-billing#studio-access-token and add a section explaining these tokens there.